inc/Engine/CriticalPath/CriticalCSSSubscriber.php

```diff
@@ -687,7 +594,7 @@ public function async_css( $buffer ) {
 		 */
 		$css_pattern = apply_filters(
 			'rocket_async_css_regex_pattern',
-			'/(?=<link[^>]*\s(rel\s*=\s*[\'"]stylesheet["\']))<link[^>]*\shref\s*=\s*[\'"]([^\'"]+)[\'"](.*)>/iU'
+			'/(?=<link[^>]*\s(rel\s*=\s*[\'"]stylesheet["\']))<link[^>]*\shref\s*=\s*[\'"]([^\'"]+)[\'"]\s*(type\s*=\s*[\'"].*[\'"])\s*(media\s*=\s*[\'"].*[\'"])(.*)>/iU'
```

Comment:
To improve readability, I would recommend to name the different capturing groups, by using the syntax `?<name>`